homeassistant/components/bmw_connected_drive/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Does not have custom services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      Does not have custom services
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      This integration doesn't have any events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      Does not have custom services
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
Personal note: I dislike `test_coordinator.py` as a whole because it assumes that the integration uses a coordinator, whilst it is only a tool. I am making this a separate comment to let this know upfront and in the next one I will explain what is wrong with the tests :)

Reply:
Not changing this, as the coordinator is the part that centrally controls entity states. If one update fails here, all entitites are unavailable. Also in the coordinator the decisions to raise issues is made.

I reworked it in #133704 to make it a clearer.